.meteor/versions

```diff
@@ -226,7 +226,6 @@ rocketchat:slackbridge@0.0.1
 rocketchat:slashcommands-archive@0.0.1
 rocketchat:slashcommands-asciiarts@0.0.1
 rocketchat:slashcommands-create@0.0.1
-rocketchat:slashcommands-create-thread@0.0.1
```

Comment:
Did you not like the slash command? I use it frequently myself

Reply:
I will reinsert :)